src/modules/assets/assets.controller.ts

```diff
@@ -66,7 +66,7 @@ export class AssetsController {
     enum: AssetHistoryEventType,
   })
   async getHistoryByAssetId(
-    @Param('id', DIDPipe) { id }: DID,
+    @Param('id', DIDPipe) { did: id }: DID,
```

Comment:
:+1: 
Can we also replace `getDidFromAddress` `https://github.com/energywebfoundation/ssi-hub/blob/f7954c7d68791cc564c70470f22f45bf341ad560/src/modules/did/did.types.ts#L107` by new DID(address).did here https://github.com/energywebfoundation/ssi-hub/blob/f7954c7d68791cc564c70470f22f45bf341ad560/src/modules/assets/assets.service.ts#L180

Reply:
@JGiter do you mean replace the getDIDFromAddress helper with the DID pipe?